tests/Doctrine/Tests/Models/Enums/Suit.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Models\Enums;
+
+enum Suit: string {
+case Hearts = 'H';
+case Diamonds = 'D';
+case Clubs = 'C';
+case Spades = 'S';
+    }
```

Comment:
CodeSniffer screws up the indentation for enums. We should deactivate the rule for files that contain enums until this has been fixed upstream.